sli/tarrayobj.cc

```diff
@@ -630,7 +630,7 @@ TokenArrayObj::operator==( const TokenArrayObj& a ) const
   // experimentally replaced by line below 090120, Diesmann
   // because [] cvx has non NULL p
   //
-  //    if( p == NULL || a.p == NULL || size() != a.size())
+  //    if( p == NULL or a.p == NULL or size() != a.size())
```

Comment:
```suggestion
  //    if( not p or not a.p or size() != a.size())
```

But probably this comment can go away completely...

Reply:
Done